.github/workflows/everything.yml

```diff
@@ -104,7 +104,7 @@ jobs:
 
   linux_ubuntu:
     needs: [format, git_checks]
-    if: needs.git_checks.outputs.num_code_changes > 0
+    if: false
```

Comment:
Can you summarize why we need to run all these builds now, even when there are no code changes?

Reply:
I have to remove these changes before merging I tmp disabled them to avoid being run.

scripts/ci/gh-actions/get-changed-files.sh consider my changes as code changes since I made changes to the readthedocs.yml